src/Native/build-native.sh

```diff
@@ -243,18 +248,29 @@ while :; do
         arm64)
             __BuildArch=arm64
             ;;
-        debug)
+        debug|-debug)
             __BuildType=Debug
             ;;
-        release)
+        release|-release)
             __BuildType=Release
             __CMakeArgs=RELEASE
             ;;
+        -osgroup*)
+             __BuildOS=`echo $1 | cut -d '=' -f2`
```

Comment:
A nit - this can be simplified to
```sh
__BuildOS=${1#*=}
```
That means - get the value $1 and delete all chars matching the glob pattern "*=".